docs/DeveloperGuide.md

```diff
@@ -571,13 +568,13 @@ The following activity diagram summarizes what happens when a user executes a cl
 ![ClearActivityDiagram](images/ClearLocationActivityDiagram.png)
 
 ### Help feature
-The help feature redirects the user to StoreMando's User Guide. If the user is connected to a internet access, StoreMando
-will redirect it's User Guide through opening another browser. Otherwise, it will have a pop out window with the User
+The help feature redirects the user to StoreMando's User Guide. If the user is connected to an internet access, StoreMando
```

Comment:
Perhaps it could be phrased as "If the device is connected to the Internet, StoreMando will automatically open the User Guide in a new browser."

Reply:
Alright, will get this changed!